components/uid.rst

```diff
@@ -158,8 +158,9 @@ following methods to create a ``Uuid`` object from it::
     $uuid = Uuid::fromBase58('TuetYWNHhmuSQ3xPoVLv9M');
     $uuid = Uuid::fromRfc4122('d9e7a184-5d5b-11ea-a62a-3499710062d0');
 
-You can also use the ``UuidFactory`` to generate UUIDs. First, you may
-configure the behavior of the factory using configuration files::
+You can also use the ``UuidFactory`` to generate UUIDs. By default, this factory
+uses UUIDv7 for default and time-based UUIDs, UUIDv5 for name-based UUIDS, and
+UUIDv4 for random-based UUIDs, but you can configure this behavior::
```

Comment:
We should probably update the config example to use other versions than 7 for the default and time based UUID version.

Reply:
Good idea! I submitted some changes. Thanks.